openshift/e2e-tests-openshift.sh

```diff
@@ -238,17 +244,36 @@ function install_knative(){
   header "Knative Installed successfully"
 }
 
-function create_serving_and_build(){
-  echo ">> Bringing up Build and Serving"
-  oc apply -f third_party/config/build/release.yaml
-  oc apply -f third_party/config/pipeline/release.yaml
-  
-  resolve_resources config/ serving-resolved.yaml $TARGET_IMAGE_PREFIX
-
-  tag_core_images serving-resolved.yaml
+function deploy_knative_operator(){
+  local COMPONENT="knative-$1"
 
-  # Remove nodePort spec as the ports do not fall into the range allowed by OpenShift
-  sed '/nodePort/d' serving-resolved.yaml | oc apply -f -
+  if oc get ns ${COMPONENT} 2>/dev/null 1>&2; then
+    echo "${COMPONENT} namespace exists - reapplying resources"
+  else
+    oc create ns ${COMPONENT}
```

Comment:
the readme does it that way:
```
apiVersion: v1
kind: Namespace
metadata:
  name: knative-serving
```

see: https://github.com/openshift/knative-serving/tree/release-v0.5.1/openshift/olm

Reply:
IMO might make sense to keep it consistent ?!

Reply:
Good catch, thanks. You are right, it makes sense to keep it consistent in one repo. I was following https://github.com/openshift-cloud-functions/knative-operators/blob/master/etc/scripts/installation-functions.sh#L259 approach.